### DISCUSSION TITLE
Styling

### DISCUSSION THREAD

**Discussion by cubecleveland** [Bunlong/react-papaparse#108] [General]
Is there any way to style the inputs and surrounding gracefully.
Using some inline div hacks. mainly it would be good if we can give it a className ?
Also, can we transfer our own Button for removing the file ?

**Comment by Bunlong** [comment]
@cubecleveland

react-papaparse v4.0.1 has been released with following features:

   * Improve styling
   * Improve code performance
   * Rewrite any existing based components to hooks

With react-papaparse v4.0.1 you can use your own button.

Please get a try!

**Please support me, If you think I'm constantly contributing to the community with cool and useful code, [please nominate me as a GitHub Star](https://stars.github.com/nominate). Thank you for your nomination, I very much appreciate it.**
